### PR TITLE
Fix/cpt

### PR DESCRIPTION
**Pull request by whaeck**
A number of corrections to produce incident charged particle ACE files:
- increasing an array size (for the number of Coulomb angles)
- correcting issue #138 in all instances (ACER and GROUPR)

I'm still working on getting a few test files added for the different cases one may encounter:
- MF6 MT2 LAW=5 LTP=1 (nuclear amplitude expansion)
- MF6 MT2 LAW=5 LTP=2 (residual cross section expansion as Legendre coefficients)
- MF6 MT2 LAW=5 LTP=12 (nuclear plus interference distribution with probabilities linear in mu)
- MF6 MT2 LAW=5 LTP=14/15 (tabulation in loglin or linlin)

Still to be added as well: release notes

### CODE REVIEW

**Comment by whaeck** [comment]
5 tests were added. As expected, only two of them changed from NJOY 2016.51 to 2016.52: the one for identical particles with spin = 0 and the one with spin = 1. The other tests (2 for different particles and one for identical particles with a half integer spin) did not change, as expected.

Release notes and test descriptions were added.

This one is good to go.

File: src/vers.f90 [hunk 1]
```diff
@@ -4,6 +4,6 @@ module version
    implicit none
    private
    character(8),public::vers='2016.52'
-   character(8),public::vday='xxNov19'
+   character(8),public::vday='02Nov19'
```
Comment (jlconlin) [inline src/vers.f90]: Did you intend for this to be tomorrow?

**Comment by jlconlin** [comment]
By the way, all the tests passed without the updated fortran files.

File: src/vers.f90 (hunk 1 above)
Comment (whaeck) [reply src/vers.f90]: Yes, 2016.51 is officially released today.

**Pull request merged**